assets/css/style.css

```diff
@@ -28,19 +28,19 @@
 *,
 *:before,
 *:after {
+  box-sizing: border-box;
```

Comment:
Many of the changes in this file are putting the properties in alphabetic order. It helps me see if styles are being duplicated more easily.

But there are instances where I removed rules or properties.